### ISSUE TITLE
Kubernetes manifest configured ILM Policy

### ISSUE THREAD

**Issue by marcstreeter** [minio/minio#12484]
**Is your feature request related to a problem? Please describe.**
Upon starting up Minio in my Kubernetes cluster I have to  download the `mc` client to apply an ILM policy on a bucket.

**Describe the solution you'd like**
I'd like the ability to specify a default ILM policy for a bucket (or all buckets) via a Kubernetes manifest.

**Describe alternatives you've considered**
I've seen examples where a bucket is created from within a Kubernetes manifest like so:

```
...
spec:
  containers:
    -image: minio...
     lifecycle:
       postStart:
         exec:
           command:
           - mkdir
           - -p
           - /data/my-bucket
     ...
```
I'm open to a solution that is just some command here that would solve the issue.  I actually assume that someone knows how to do it this way but I can't find any documentation/community-chat about it. I thought to look at the [configuration guide](https://docs.min.io/docs/minio-server-configuration-guide.html) but also didn't see anything listed as a solution.

**Additional context**
Seems related in part to #4769 but the comments there still [use mc as a separate container](https://github.com/minio/minio/issues/4769#issuecomment-526622661)

**Comment by harshavardhana** [comment]
@marcstreeter this will be fixed by #12509

**Comment by marcstreeter** [comment]
🥳🥳🥳

**Comment by harshavardhana** [comment]
> @marcstreeter this will be fixed by #12509

This is merged
